src/terraform/command_helpers.go

```diff
@@ -14,79 +14,79 @@ import (
 
 // CompileVarFlags compiles the variable flags passed into a Terraform command
 func CompileVarFlags(deployConfig deploy.Config, secrets types.Secrets, imagesMap map[string]string) ([]string, error) {
```

Comment:
For testing purposes, thoughts on having a `GetVarMap()` function which builds the entire var map and this function calls that and converts it to flags. That way we can test that function instead of the individual functions its calls.